projects/uitdatabank/docs/entry-api/school-events.md

```diff
@@ -0,0 +1,356 @@
+
+* level 1 label `cultuurkuur_Secundair-onderwijs`
+* level 2 label `cultuurkuur_Voltijds-gewoon-secundair-onderwijs`
+
+| Education level                                       | Label (level 3)                                                   |
+| :---------------------------------------------------- | :---------------------------------------------------------------- |
+| Eerste graad                                          | `cultuurkuur_eerste-graad`                                        |
+| Tweede graad                                          | `cultuurkuur_tweede-graad`                                        |
+| Derde graad                                           | `cultuurkuur_derde-graad`                                         |
+| Secundair na Secundair (Se-n-Se)                      | `cultuurkuur_Secundair-na-secundair-(Se-n-Se)`                    |
+| Onthaalonderwijs voor anderstalige nieuwkomers (OKAN) | `cultuurkuur_Onthaalonderwijs-voor-anderstalige-nieuwkomers-OKAN` |
+
+##### Level 4 labels
+
+> A level 4 label must always be combined with both a level 1, level 2 and level 3 label.
+
+**Gewoon kleuteronderwijs**
+
+The following level 4 labels must always be combined with
+
+* level 1 label `cultuurkuur_basisonderwijs`
+* level 2 label `cultuurkuur_Gewoon-basisonderwijs`
+* level 3 label `cultuurkuur_Gewoon-kleuteronderwijs`
+
+| Education level    | Label (level 4)                |
+| :----------------- | :----------------------------- |
+| Kleuter (2-3 jaar) | `cultuurkuur_Kleuter-2-3-jaar` |
+| Kleuter (3-4 jaar) | `cultuurkuur_Kleuter-3-4-jaar` |
+| Kleuter (4-5 jaar) | `cultuurkuur_Kleuter-4-5-jaar` |
+
+**Gewoon lager onderwijs**
+
+The following level 4 labels must always be combined with
+
+* level 1 label `cultuurkuur_basisonderwijs`
+* level 2 label `cultuurkuur_Gewoon-basisonderwijs`
+* level 3 label `cultuurkuur_Gewoon-lager-onderwijs`
+
+| Education level | Label (level 4)          |
+| :-------------- | :----------------------- |
+| Eerste graad    | `cultuurkuur_1ste-graad` |
+| Tweede graad    | `cultuurkuur_2de-graad`  |
+| Derde graad     | `cultuurkuur_3de-graad`  |
+
+**Eerste graad secundair**
+
+The following level 4 labels must always be combined with
+
+* level 1 label `cultuurkuur_Secundair-onderwijs`
+* level 2 label `cultuurkuur_Voltijds-gewoon-secundair-onderwijs`
+* level 3 label `cultuurkuur_eerste-graad`
+
+| Education level               | Label (level 4)                              |
+| :---------------------------- | :------------------------------------------- |
+| Eerste leerjaar A             | `cultuurkuur_1ste-leerjaar-A`                |
+| Eerste leerjaar B             | `cultuurkuur_1ste-leerjaar-B`                |
+| Tweede leerjaar               | `cultuurkuur_2de leerjaar`                   |
+| Beroepsvoorbereidend leerjaar | `	cultuurkuur_beroepsvoorbereidend-leerjaar` |
+
+**Tweede graad secundair**
+
+The following level 4 labels must always be combined with
```

Comment:
Sidenote and not blocking this guide from being published: I'm seeing this being repeated a lot and it feels like something that we could implement in UDB3 itself? It feels a bit redundant to ask an integrator to add lvl 1, 2 and 3 labels that we can easily derive ourselves from the level 4 label.

Reply:
It was a conscious decision not to add the custom logic for Cultuurkuur in UiTdatabank, because there is too little overlap between the goals of Cultuurkuur and the goals of UiTdatabank. 

If you enter a school event on www.uitdatabank.be a user is notified on the preview page that he has to enrich his event on Cultuurkuur (see `https://www.uitdatabank.be/event/49bf2f7f-9ffb-4ef9-b110-6495b85ca06e/preview` for example). The logic of the labels is mostly handled by the user interface on www.cultuurkuur.be: if you add a level 1 label, you'll be able to select the corresponding level 2, level 3 and level 4 labels.

Reply:
Okay makes sense. Just to clear, I meant on the API though. So if an integrator would send us an event with a level 4 label but not level 3/2/1 labels, we could technically add those automatically. But that's maybe only a solution to a very small group of integrators and not worth it. At least it is clear from the guide I think what is needed!

Reply:
Yes, indeed it would be to a very small group of integrators. And it would be hard to maintain I think, since the labels for Cultuurkuur change over time (and thus the logic between the levels) - see my comment about "onderwijsvernieuwing".

Reply:
Alright!